code/game/objects/items/devices/modkit.dm

```diff
@@ -147,3 +147,41 @@
 	parts[1] =	1
 	original[1] = /obj/item/weapon/gun/energy/gun
 	finished[1] = /obj/item/weapon/gun/energy/gun/nuclear
+
+
+/obj/item/device/modkit/paintkit
+	name = "welding helmet stencil kit"
+	desc = "A kit for automatically painting welding helmets with preset designs"
+	icon_state = "modkit"
+
+
+	var/obj/item/device/modkit/paintkit/active_paint = /obj/item/clothing/head/welding/engie
+
+	var/paints = list("engineering" = /obj/item/clothing/head/welding/engie,
+	"carp" = /obj/item/clothing/head/welding/carp,
+	"fancy" = /obj/item/clothing/head/welding/fancy,
+	"knightly" = /obj/item/clothing/head/welding/knight,
+	"demonic" = /obj/item/clothing/head/welding/demon,
+	"fiery" = /obj/item/clothing/head/welding/fire,
+	"red" = /obj/item/clothing/head/welding/red,
+	"white" = /obj/item/clothing/head/welding/white,
+	"blue" = /obj/item/clothing/head/welding/blue)
+
+
+/obj/item/device/modkit/paintkit/attack_self(mob/user)
+	if(paints)
```

Comment:
Can this ever fail without it being a bug?

Reply:
elaborate. what can fail? should it be able to fail or not?

Reply:
You're checking whether `paints` exists but there's no reason for it *not* to exist, so if anything this is only gonna make finding bugs harder if something breaks.